catalog-creamy-ice.html

```diff
@@ -16,7 +16,14 @@
           </a>
           <ul class="navigation-main">
             <li class="navigation-main-item">
-              <a class="navigation-main-link" href="">Каталог</a>
+              <a class="navigation-main-link navigation-link-catalog" href="">
+                <span class="navigation-text">Каталог</span>
+                <svg class="navigation-icon" aria-hidden="true" focusable="false" width="12" height="7"
+                  viewBox="0 0 12 7" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd" d="M1.8 1.05A.67.67
+                0 0 0 .88 2l4.66 4.66a.66.66 0 0 0 .95 0L11.14 2a.67.67 0 0 0-.94-.95L6 5.25l-4.2-4.2Z">
```

Comment:
Есть проблемы с валидацией HTML (проверьте).
![image](https://github.com/htmlacademy-htmlcss/1162653-gllacy-1/assets/17790771/6e47002e-e50c-480c-9bd5-db5651a13f08)
Проблема в том, что у тега `path` не стоит слеш в конце:
```suggestion
                  <path fill-rule="evenodd" clip-rule="evenodd" d="M1.8 1.05A.67.67
                0 0 0 .88 2l4.66 4.66a.66.66 0 0 0 .95 0L11.14 2a.67.67 0 0 0-.94-.95L6 5.25l-4.2-4.2Z"/>
```
На главной странице точно также.